CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 ## History
 
 - [Unreleased](#unreleased)
+- [v2.9.3](#v293)
```

Comment:
could you please exclude this from this PR?
it is unrelated, I will open a new PR to fix this.

Reply:
https://github.com/kedacore/keda/pull/4264

Reply:
sure, omw

Reply:
done :)